### ISSUE TITLE
Compatibility with ISO14443 Type 4B NFC tags?

### ISSUE THREAD

**Issue by manojrohit** [michaelneurohr/NfcReaderLibrary#1]
How can I change this Code /Is there any other code compatible with ISO 14443 Type 4B nfc tags?

**Comment by michaelneurohr** [comment]
Yes. You can download the NFC Reader Library from http://www.nxp.com/pages/:NFC-READER-LIBRARY
It supports Type B besides others. To use it with EXPLORE-NFC, please download "NFC Reader Library for Linux" version 4.040.05.011646. It is available under "previous" versions.

**Comment by manojrohit** [comment]
I could only find phpalI14443p3a, phpalI14443p3b, phpalI14443p4 and phpalI14443p4a Header files. I cannot find phpalI14443p4b in the nxpnfcrdLib Folder.
